Userland/Libraries/LibWeb/HTML/Window.cpp

```diff
@@ -1105,20 +1105,16 @@ bool Window::confirm(Optional<String> const& message)
     // FIXME: Make this fully spec compliant.
     // NOTE: `message` has an IDL-provided default value and is never empty.
     if (auto* page = this->page())
-        return page->did_request_confirm(message->to_deprecated_string());
+        return page->did_request_confirm(*message);
     return false;
 }
 
 // https://html.spec.whatwg.org/multipage/timers-and-user-prompts.html#dom-prompt
 Optional<String> Window::prompt(Optional<String> const& message, Optional<String> const& default_)
 {
     // FIXME: Make this fully spec compliant.
-    if (auto* page = this->page()) {
-        auto response = page->did_request_prompt(message->to_deprecated_string(), default_->to_deprecated_string());
-        if (response.is_null())
-            return {};
-        return String::from_deprecated_string(response).release_value_but_fixme_should_propagate_errors();
-    }
+    if (auto* page = this->page())
+        return page->did_request_prompt(*message, *default_);
```

Comment:
🤯

Reply:
@konradekk Please refrain from posting nonsensical comments like these, we need a review instead!

Reply:
i just really liked what it became (seeing the previous version side-by-side).

but sure—got that. 😣